### DISCUSSION TITLE
Error 22 file not found. Most likely one of the plot folders in your config.yaml has an issue

### DISCUSSION THREAD

**Discussion by apicoypala** [Chia-Network/chia-blockchain#5672] [Support]
Por favor, necesito ayuda con este problema.. Alguien sabe como solucionar esto?

Please, I need help with this problem .. Anyone know how to solve this?

**Comment by lvcivs** [comment]
As the message says, you need to check the`config.yaml` file on your system. You probably have an invalid directory specified there.

**Comment by BrandtH22** [comment]
@apicoypala , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
